Add default location for chrony.conf on Ubuntu

kharry5 opened the pull request:
### What does this PR do?
This PR adds the Ubuntu default location for chrony.conf to the list of locations returned by getLocalDefinedNTPServers

### Motivation
Without this change the agent will fail to find the ntp config when using chrony on Ubuntu

kharry5 committed:
Add default location for chrony.conf on Ubuntu

On Ubuntu systems the compiled-in default location for chrony.conf is /etc/chrony/chrony.conf

kharry5 commented:
Is there anything else I need to do to get this approved / merged?

NouemanKHAL commented:
Hi @kharry5, Thanks for your contribution!

Looks good to me! We just need to add a release note, and run the rest of the workflows before merging.

For reference, please check the [CONTRIBUTING.md](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md) document which provides the guidelines for contributing to this repository.

Thanks!

kharry5 committed:
Add release notes

kharry5 commented:
@NouemanKHAL I've added the release notes, but I've added them from my work laptop that didn't have a real email address configured so now bits-bot is complaining about the CLA not being signed by that user.
Is there a way to correct this for this PR or should I close this and create a new one?

NouemanKHAL commented:
@kharry5 looks fine to me, could you update your branch with main? Feel free to rebase or merge the main branch, either works.

NouemanKHAL commented:
@kharry5 It seems that the commit history has become messy, could you rebase your branch with main and (force) push again?

You can refer to the steps documented [here](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#after-the-first-review).

kharry5 commented:
@NouemanKHAL I've attempted the rebase + force push. If that didn't do the trick then I'll just create a new PR.

NouemanKHAL commented:
/merge
